packages/near-fast-auth-signer/src/components/SignMultichain/utils.ts

```diff
@@ -218,24 +281,25 @@ export const multichainSignAndSend = async ({
 };
 
 export const multichainGetFeeProperties = async ({
-  asset,
+  chain,
   to,
   value,
-  from = null,
+  from
 }: {
-  asset: Chain;
+  chain: number;
   to: string;
   value: string;
-  from?: string;
+  from: string;
 }) => {
-  if (asset === 'BTC') {
+  // chain is the slip044 chain id
+  if (chain === 0) {
```

Comment:
Same as the previous comment, properly type it to get errors in case we add/remove chain and forget to update here